app.js

```diff
@@ -25,14 +25,23 @@ connectDB()
 
 const app = express();
 
+// ** Body parser 
```

Comment:
Parse incoming request bodies in a middleware before your handlers, available under the req.body property.